static/js/components/CommentForms_test.js

```diff
@@ -33,7 +36,9 @@ describe("CommentForms", () => {
   const renderPostForm = (props = {}) =>
     mount(
       <Provider store={helper.store}>
-        <ReplyToPostForm post={post} {...props} />
+        <Router store={helper.store} history={helper.browserHistory}>
+          <ReplyToPostForm post={post} {...props} />
+        </Router>
```

Comment:
why is the `<Router>` necessary here?

Reply:
Without it, this error is thrown due to `<Link>` being in the `LoginPopup`:
```
Invariant Violation: You should not use <Link> outside a <Router>
```

Reply:
gotcha